tests/integration/synapseclient/test_command_line_client.py

```diff
@@ -82,9 +82,9 @@ def test_command_line_client(test_state):
                  'synapse',
                  '--skip-checks',
                  'create',
-                 '-name',
+                 '--name',
                  str(uuid.uuid4()),
-                 '-description',
+                 '--description',
```

Comment:
Does PEP8 have a standard on CL arguments?  I see what you're doing here, just wondering.

Reply:
Source from here as command line standards: https://en.wikipedia.org/wiki/Command-line_interface#Arguments